src/index.html

```diff
@@ -1,12 +1,266 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#home"><img class="header__logo" src="images/logo.png" alt="logo"></a>
```

Comment:
two tags in one line, also three attributes in one row, you should fix it

Reply:
```suggestion
      <a href="#home">
        <img
          class="header__logo"
          src="images/logo.png"
          alt="logo"
        >
      </a>
```